src/views/Accessibility/TestingTemplates/index.tsx

```diff
@@ -0,0 +1,180 @@
+        </li>
+        <li>
+          <Link href="#remediation-plan">
+            {t('testingTemplates.remediationPlanSection.heading')}
+          </Link>
+        </li>
+      </ul>
+    </div>
+  );
+
+  const testPlanSection = (
+    <div>
+      <h2 id="test-plan">{t('testingTemplates.testPlanSection.heading')}</h2>
+      <p>{t('testingTemplates.testPlanSection.description')}</p>
+      <ul className="accessibility-testing-templates__test-plan-list">
+        {testPlanList.map(item => (
+          <li key={item}>{item}</li>
+        ))}
+      </ul>
+    </div>
+  );
+  const remediationPlanSection = (
+    <div>
+      <h2 id="remediation-plan">
+        {t('testingTemplates.remediationPlanSection.heading')}
+      </h2>
+      <p>{t('testingTemplates.remediationPlanSection.description')}</p>
+      <ul className="accessibility-testing-templates__remediation-plan-list">
+        {remediationPlanList.map(item => (
+          <li key={item}>{item}</li>
+        ))}
+      </ul>
+    </div>
+  );
+
+  const downloadVPAT = (
+    <div className="accessibility-testing-templates__downloadBox">
+      <h3 className="margin-top-0">
+        {t('testingTemplates.vpatSection.subSection.downloadVPAT.heading')}
+      </h3>
+      <p>
+        <Link
+          href="https://www.itic.org/policy/accessibility/vpat"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {t(
+            'testingTemplates.vpatSection.subSection.downloadVPAT.line1.linkText'
+          )}
+        </Link>
+        {` `}
+        {t(
+          'testingTemplates.vpatSection.subSection.downloadVPAT.line1.otherText'
+        )}
+      </p>
+      <p className="display-flex flex-row flex-align-center accessibility-testing-templates__alert-note">
+        <i className="fa fa-exclamation-circle margin-right-1" />
+        {` `}
+        {t('testingTemplates.vpatSection.subSection.downloadVPAT.line2.text')}
+      </p>
+      <p>
+        <Link
+          href="https://www.youtube.com/watch?v=kAkSV9xiJ1A"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {t(
+            'testingTemplates.vpatSection.subSection.downloadVPAT.line3.linkText'
+          )}
+        </Link>
+        {` `}
+        {t(
+          'testingTemplates.vpatSection.subSection.downloadVPAT.line3.otherText'
+        )}
+      </p>
+    </div>
+  );
+
+  const vpatSection = (
+    <div>
+      <h2 id="vpat">{t('testingTemplates.vpatSection.heading')}</h2>
+      <p>{t('testingTemplates.vpatSection.description')}</p>
+      <h3>{t('testingTemplates.vpatSection.subSection.heading')}</h3>
+      <ul className="accessibility-testing-templates__vpat-list">
+        <li>{t('testingTemplates.vpatSection.subSection.item1.text')}</li>
+        <div className="padding-left-2">
+          <dl>
+            {vpatConformanceLevels.map(level => (
+              <div>
+                <dt className="text-bold display-inline">{level.name}</dt>{' '}
```

Comment:
Default behavior for dt/dd is for the dd to start on the next line, indented. To get the dt and dd aligned on the same line, I made them each `display: inline` and put them inside a block.  Is this an ok way to do that?

Reply:
Yeah, styling can be difficult because of the default styling on some of the semantic elements. You're on the right track tho!

The `dl` needs to be inside of the `li` because `ol` can only have `li` as children.

 You'll also need to update the snapshot after 😬

Reply:
Sorry one more thing!

You can also add a `title` to the `dl` to give better context on what the definition list is for.
For example, `<dl title="Conformance levels">`